lib/addons/index.ts

```diff
@@ -31,6 +31,7 @@ export * from './ebs-csi-driver';
 export * from './efs-csi-driver';
 export * from './istio-base';
 export * from './istio-control-plane';
+export * from './cert-manager';
```

Comment:
let's attempt to use alphabetical order in imports.

Reply:
done